tools/clang-format.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+cd `dirname $0`/..
+IGNORE_FILES="src/misc/warnings.hpp"
+
+ERROR=0
+for FILE in `find src -iname "*.hpp" -or -iname "*.cpp"`; do
```

Comment:
What about the c++ files in `qiskit_aer` and `test`?

Reply:
good comment. let me add them into the list. maybe `contrib` should be included.